src/App.jsx

```diff
@@ -19,72 +24,28 @@ function Home() {
   );
 }
 
-function Products() {
-  const { data, loading, error } = useFetch('products');
-  if (loading) return <div>Loading...</div>;
-  if (error) return <div>{error.message}</div>;
-
-  return (
-    <div>
-      <h1 class="lsheading">📜Item List📜</h1>      
-      <div class="list">
-        {data.map(product => (
-          <div class="itemnum" key={product.id}>
-            <NavLink to={`/products/${product.id}`}>
-              <img src={product.image} alt={product.title} />
-              <p>{product.title}</p>
-            </NavLink>
-          </div>
-        ))}
-      </div>
-    </div>
-  );
-}
 
-const ProductDetail = () => {
-  const { productId } = useParams();
-  const history = useHistory();
-  const { isLoading, data: product, error } = useFetch(`/api/products/${productId}`);
-
-  // Handle error
-  if (error) {
-    return <div>Error: {error.message}</div>;
-  }
-
-  // Render loading indicator if data is still loading
-  if (isLoading || !product) {
-    return <LoadingIndicator />;
-  }
-
-  return (
-    <div>
-      <button onClick={() => history.goBack()}>Назад</button>
-      <h1>{product.title}</h1>
-      <img src={product.image} alt={product.title} />
-      <p>{product.description}</p>
-    </div>
-  );
-};
 
 function App() {
+  
   return (
     <Router>
```

Comment:
The navigation bar should always persist. The loading state shouldn't take over the entire screen.

Look at the Outlet section of https://neit.instructure.com/courses/56465/pages/live-coding-adding-react-router-to-week-2?module_item_id=2530831 . This is where we covered it.